Update distribution.md (#496)
codemagic-ci-cd/codemagic-docs: content/yaml-publishing/distribution.md

### Google Play

> Codemagic enables you to automatically publish your app either to one of the predefined tracks (`internal`- publish for internal testing and QA, `alpha`- publish for testing with a small group of trusted users, `beta`- publish for testing to a wider set of users and `production`- release the app to production) on Google Play or to your custom closed testing tracks. In order to do so, you will need to set up a service account in Google Play Console and add the `JSON` key file to your Codemagic configuration file; see how to [set up a service account](../knowledge-base/google-play-api/). The proper way to add your keys to `codemagic.yaml` is to `Secure` the key file contents in the Environment Variables section.

Reviewer: ```suggestion
Codemagic enables you to automatically publish your app either to one of the predefined tracks (`internal`- publish for internal testing and QA, `alpha`- publish for testing with a small group of trusted users, `beta`- publish for testing to a wider set of users and `production`- release the app to production) on Google Play or to your custom closed testing tracks. In order to do so, you will need to [set up a service account in Google Play Console](../knowledge-base/google-play-api/) and save the contents of the `JSON` key file as a [secure environment variable](../variables/environment-variable-groups/#storing-sensitive-valuesfiles) in application or team settings.

If your application supports [in-app updates](https://developer.android.com/guide/playcore/in-app-updates), Codemagic allows setting the update priority. Otherwise, `in_app_update_priority` can be omitted or set to `0`.

In addition, Codemagic supports [staged releases](https://support.google.com/googleplay/android-developer/answer/6346149?hl=en), allowing users to choose which fraction of the testers or users get access to the application. To release to everyone, omit `rollout_fraction` from codemagic.yaml. 

```yaml
publishing:
  google_play:                                             # For Android app

>     credentials: $GCLOUD_SERVICE_ACCOUNT_CREDENTIALS       # JSON key file for Google Play service account secured in Environment Variables

Reviewer: ```suggestion
    credentials: $GCLOUD_SERVICE_ACCOUNT_CREDENTIALS       # JSON key file for Google Play service account saved as a secure environment variable
```

## Publishing an app to Firebase App Distribution

> If you use a Firebase service, secure `google-services.json` as `ANDROID_FIREBASE_SECRET` environment variable for Android

Reviewer: ```suggestion
If you use a Firebase service, save `google-services.json` as a secure environment variable `ANDROID_FIREBASE_SECRET` for Android
```
